aleph-client/src/contract/util.rs

```diff
@@ -46,3 +46,21 @@ pub fn to_account_id(value: Value) -> Result<AccountId> {
         _ => Err(anyhow!("Expected {:?} to be a string", value)),
     }
 }
+
+/// Returns `Ok(bool)` if the given `Value` represents one, or `Err(_)` otherwise.
+/// ```
+/// # #![feature(assert_matches)]
+/// # use std::assert_matches::assert_matches;
+/// # use anyhow::anyhow;
+/// # use aleph_client::contract::util::to_bool;
+/// use contract_transcode::Value;
+///
+/// assert_matches!(to_bool(Value::Bool(true)), Ok(true));
+/// assert_matches!(to_bool(Value::UInt(42)), Err(_));
+/// ```
+pub fn to_bool(value: Value) -> Result<bool> {
```

Comment:
It'd be more "rusty" to implement [TryFrom](https://doc.rust-lang.org/stable/std/convert/trait.TryFrom.html) fro such a conversion and then use .into() for a short & sweet syntax

Reply:
Can I implement `TryFrom` for a type from another crate?

Reply:
yes, kinda: https://doc.rust-lang.org/rust-by-example/generics/new_types.html

Reply:
Yeah, I knew about that, just thought it would be too much cruft. Hm, I guess it could be something like:

```rust
// Instead of
to_bool(result)
/// do
ValueConvert(result).into()
```

at the call site? Honestly, I don't think this is very clean, but can change if you think it's better, let me know.

Reply:
I do not have a problem with these conversions in their current form.